assets/inference/environments/mlflow-ubuntu22.04-py312-cpu-inference/context/conda_dependencies.yaml

```diff
@@ -6,4 +6,8 @@ dependencies:
 - python=3.12
 - pip
 - pip:
-  - azureml-inference-server-http=={{latest-pypi-version}}
\ No newline at end of file
+  - azureml-inference-server-http=={{latest-pypi-version}}
+  - azureml-ai-monitoring=={{latest-pypi-version}}
+  - numpy
```

Comment:
azureml-ai-monitoring depends on pandas which depends on numpy, if you have them here I would recommend to pin it

Reply:
Updated! Added pandas.

Reply:
what I meant, you don't need numpy nor pandas

Reply:
Updated!